css/style.css

```diff
@@ -686,6 +1721,43 @@ body {
   }
 
   .product-card {
-    min-width: 264px;
+    min-width: 247px;
+  }
+
+  .product-card__img {
+    padding-left: 95px;
+  }
+
+.product-card__discr {
+    padding-left: 29px;
+}
+
+.product-card__price {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5612618)